src/Analyzers/CSharp/Analyzers/UseUTF8StringLiteral/UseUTF8StringLiteralDiagnosticAnalyzer.cs

```diff
@@ -49,10 +49,12 @@ protected override void InitializeWorker(AnalysisContext context)
                 if (!context.Compilation.LanguageVersion().IsCSharp11OrAbove())
                     return;
 
+                if (context.Compilation.GetTypeByMetadataName(typeof(ReadOnlySpan<>).FullName!) is null)
```

Comment:
I don't recall the details. But there was bugs previously due to the use of `GetTypeByMetadataName`.


```suggestion
                if (context.Compilation.GetBestTypeByMetadataName(typeof(ReadOnlySpan<>).FullName!) is null)
```